### PR TITLE
1681: Add twig template processing for email body and confirmation me…

### PR DESCRIPTION
**Pull request by robertromore**
…ssage

### CODE REVIEW

File: springboard_twig/springboard_twig.module [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    }
+    catch (Twig_Error_Syntax $e) {
+      drupal_set_message($e->getMessage(), 'error');
+    }
+    catch (Exception $e) {
+      drupal_set_message($e->getMessage(), 'error');
+    }
+  }
+  else {
+    $error = $library['error'];
```
Comment (robertromore) [inline springboard_twig/springboard_twig.module]: Thanks! I've made those changes.

File: springboard_twig/springboard_twig.module [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    '!types' => $available_types,
+    '!url' => l(t('the Twig project page'), 'http://twig.sensiolabs.org/doc/1.x/', array(
+      'attributes' => array(
+        'target' => '_blank',
+      ),
+    )),
+    '!breaks' => '<br/><br/>',
+  ));
+
+  drupal_alter('springboard_twig_form_twig_info_field_description', $description_markup);
```
Comment (scottatdrake) [inline springboard_twig/springboard_twig.module]: Could you add this to springboard_twig.api.php?

File: springboard_twig/springboard_twig.module [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+
+  $twig_description = array(
+    'twig' => array(
+      '#type' => 'markup',
+      '#prefix' => '<br />',
+      '#markup' => $description_markup,
+    ),
+  );
+
+  drupal_alter('springboard_twig_form_twig_info_field_variables', $type, $twig_variables);
```
Comment (scottatdrake) [inline springboard_twig/springboard_twig.module]: Could you add this to springboard_twig.api.php?

File: springboard_twig/springboard_twig.module [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+
+      $time = time();
+      $replacements['date'] = array(
+        'long' => format_date($time, 'long'),
+        'medium' => format_date($time, 'medium'),
+        'short' => format_date($time, 'short'),
+      );
+
+      // Let other modules add or replace their own variables.
+      drupal_alter('springboard_twig_variable_replacements', $node_type, $replacements);
```
Comment (scottatdrake) [inline springboard_twig/springboard_twig.module]: Could you add this to springboard_twig.api.php?

File: springboard_twig/springboard_twig.module [hunk 5]
```diff
@@ -0,0 +1,212 @@
+<?php
```
Comment (scottatdrake) [inline springboard_twig/springboard_twig.module]: Looks like this was fixed.

**Pull request merged**